src/utils/time-entry-utils/timeEntryUtils.js

```diff
@@ -37,14 +40,11 @@ export const removeTimecardContextEntry = (
   setTimeEntries(newTimeEntries);
 };
 
-export const calculateFinishTimeOnNextDay = (
-  startTimeValue,
-  finishTimeValue
-) => {
+export const isFinishTimeOnNextDay = (startTimeValue, finishTimeValue) => {
   if (startTimeValue && finishTimeValue) {
-    return Math.floor(
-      parseFloat(startTimeValue.replace(/:/g, '.')).toFixed(2) /
-        parseFloat(finishTimeValue.replace(/:/g, '.')).toFixed(2)
-    );
+    return dayjs(
+      dayjs().format('YYYY-MM-DD') + '' + finishTimeValue
+    ).isSameOrBefore(dayjs(dayjs().format('YYYY-MM-DD') + '' + startTimeValue));
```

Comment:
I think this can be further cut down to the following by using the existing formatDate util function

```suggestion
    return dayjs(formatDate(dayjs()) + finishTimeValue).isSameOrBefore(
      dayjs(formatDate(dayjs()) + startTimeValue)
    );
```